Pinta.Core/EventArgs/ToolMouseEventArgs.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cairo;
+using Gdk;
+
+namespace Pinta.Core
+{
+	public class ToolMouseEventArgs : EventArgs
+	{
+		/// <summary>
+		/// Specifies whether the Alt key is currently pressed.
+		/// </summary>
+		public bool IsAltPressed => State.HasFlag (ModifierType.Mod1Mask);
+
+		/// <summary>
+		/// Specifies whether the Control key is currently pressed.
+		/// </summary>
+		public bool IsControlPressed => State.HasFlag (ModifierType.ControlMask);
```

Comment:
I'm thinking this could check `MetaMask` instead on macOS, since in the GTK3 build we're now using Cmd for the menu item shortcuts instead of Ctrl. I had been planning to look into doing this in `IsControlPressed()` and some of the tools that directly check with `ControlMask`

Reply:
Sounds good, I will definitely defer to you on making Pinta feel better on Mac.  😄 

One other thing to adjust is our tooltips that say "Ctrl", like `LineCurveTool`.

Reply:
Ah, good point about the tooltips! I'll make a note to fix that.